scripts/Thaumic-Horizons.zs

```diff
@@ -26,19 +26,19 @@ mods.thaumcraft.Infusion.removeRecipe(<ThaumicHorizons:recombinator>);
 
 
 // --- Planar Conduit
-mods.thaumcraft.Arcane.addShaped("planarTheory", <ThaumicHorizons:planarConduit>, "aer 100, aqua 100, ignis 100, terra 100, ordo 100, perditio 100", [
-[<ore:plateShadow>, <ore:plateEnderEye>, <ore:plateShadow>],
+mods.thaumcraft.Arcane.addShaped("planarTheory", <ThaumicHorizons:planarConduit>, "aer 50, aqua 50, ignis 50, terra 50, ordo 50, perditio 50", [
```

Comment:
no one said to change that to 50

Reply:
I did in the issue, I realised later.

Reply:
Wait i guess the primal pearl in the middle need more than 50 vis so why we change that to 50 ?

Reply:
Primal charm is 50. If the planar conduit costs 100 vis, you need an augmented greatwood scepter (vacuum freezer), or high level caps (forbidden research or titanium)
![image](https://user-images.githubusercontent.com/17787093/103467761-20508a80-4d4a-11eb-914e-7fe310eda576.png)

Reply:
What about 75? Then you need an iron capped greatwood scepter at least?

Reply:
yea make it 75